loopFibonacci.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+int main()
+{
+	printf("How many numbers in the fibonacci series do you want to print?\n");
+	int k;
+	scanf("%d", &k);
+	int first = 1, second = 1, sum;
+	printf("1 1 ");
```

Comment:
What about if the user only wants 0 or 1 of the fibonacci numbers?? Also, where are your comments?